src/SW_netCDF.c

```diff
@@ -4731,6 +4768,21 @@ void SW_NC_read_out_vars(
                 continue;
             }
 
+            if (Str_CompareI(
+                    (char *) SWVarUnits[currOutKey][varNum], input[SWUnitsInd]
+                ) != 0) {
+                LogError(
+                    LogInfo,
+                    LOGWARN,
+                    "%s: Found an input variable (%s) with a 'SW2 units' "
+                    "value that does not match the unit SOILWAT2 uses."
```

Comment:
We need a space between the sentences in the warning message. Otherwise, it looks great!